docs/concepts/account-linking/index.mdx

```diff
@@ -3,80 +3,120 @@ title: Account Linking (FLIP 72)
+        signer.inbox.publish(capability, name: "accountCapA", recipient: 0x1)
+    }
+}
+```
+
+****************************Claim capability****************************
+
+The account A claims the capability published by account B.
+
+```json
+transaction {
+    prepare(signer: AuthAccount) {
+        let capability = signer.inbox.claim<&AuthAccount>("accountCapB", provider: 0x2)!
+        let accountRef = capability.borrow()!
+    }
+}
+```
+
+## How is account linking useful?
+
+Account linking is very useful in the case of onboarding non-existing Flow users to your Flow application. Because participating on Flow network requires having a Flow account, which a non-web3-native user will struggle to achieve a lot of applications create the account for the user.
+
+The problem with this approach becomes when a user wants to use that account with another application, or move the items stored in that account out of the application. With account linking it’s possible for the application to create the account instead of the user, but still allow for the user to later link that account to any other accounts he will own.
+
+Account linking allows having multiple Flow accounts and abstracts them as a single account.
 
 All assets in the app account can now jump the walled garden to play in the rest of the Flow ecosystem. No transactions are needed to move assets from the child account to the parent account, as the parent account already has full access to the assets in the child account.
 
-</Callout>
+![Multiple parent-child accounts on Flow](resources/multiple-accounts.png)
+
+This shared control over the digital items in the in-app account enables users to establish real ownership of the items beyond the context of the app, where they can use their parent account to view inventory, take the items to other apps in the ecosystem, such as a marketplace or a game.
+
+Most importantly, users are able to do this without the need to transfer the digital items between accounts, making it seamless to continue using the original app while also using other apps.
+
+## Security Consideration with Account Linking
+
+Account linking is a very powerful feature of Cadence and thus it must be treated with care. In this document, we’ve always talked about account linking between two accounts we own, even if the child account is managed by a third-party application, that we access through an authentication mechanism, it’s still treated as our account.
+
+Creating an `AuthAccount` capability and publishing it to an account we don’t own means we are giving that account full access to our account. This should be seen as an anti-pattern.
+
+<aside>
+❗ Creating an `AuthAccount` capability and sharing it with another person account is in concept the same as giving that person your accounts private keys.
+
+</aside>
+
+Because pure account linking can be dangerous Flow introduces a Hybrid Custody contract that helps manage child accounts and access.
```

Comment:
```suggestion
Because pure account linking can be dangerous, Flow introduces the `HybridCustody` contract that helps custodial applications regulate access and parent account manage their many child accounts.
```